### ISSUE TITLE
Please show base by default in the 3-way-merge editor

### ISSUE THREAD

**Issue by papb** [microsoft/vscode#165628]
Since #155277 we can now [see the base](https://github.com/microsoft/vscode/issues/155277#issuecomment-1283623303), but it is turned off by default. Please turn it on by default. When resolving a conflict, seeing the base (in a panel for it that is horizontally aligned with the other two) is vital.

Some people might even make mistakes while resolving conflicts because they don't understand well how git conflicts work and didn't even think that the "show base" option exists.

Related:
- #155277
- #159920
- #155251
- #154877

Note: I've seen people advocating for the "show base" option to not even exist, and make it show always, impossible to turn off. But let's save that to another issue.

Pinging some people that might be interested: @ruffin--, @yairchu, @jesperkristensen

**Comment by hediet** [comment]
Currently only ~0.1% of users open the merge editor with the base view visible.
While I agree that the base view is very useful, many people don't think of combining changes, but rather of combining states when solving merge conflicts. To them, the base view is not as useful.

I suggest the following: We can explore how to make the base view option more prominent and mention that view in the docs. Once a significant userbase turns on the base view, we can think of changing the default.

**Comment by yairchu** [comment]
> Currently only ~0.1% of users open the merge editor with the base view visible.

Yes, most users don't change the defaults.

As another example, 99% of people have never opened vscode. Not even once.
Can we somehow infer from that that vscode is unnecessary? After all many people don't think of editing textual files, but other stuff people often do.

**Comment by papb** [comment]
@hediet 

> Many people don't think of combining changes, but rather of combining states when solving merge conflicts.

Indeed, many people don't think of combining changes, but rather of combining states when solving merge conflicts. However, that thought is inherently wrong, confusing and unhelpful, as mentioned in the linked issues.

> To them, the base view is not as useful.

I disagree. The base view would be specially useful to them, forcing them to think correctly. Git is hard, and helping people do the correct think by design is the best approach.

**Comment by jastich** [comment]
I didn't even know it was possible to open the base view until very recently.  So I guess I was one of the 99.9% who didn't find it useful before?

The base view is essential for a proper merge.  Particularly in large codebases.

**Comment by andrew-uoa** [comment]
I also didn't know base view was available until now. In the past I used base view in other merge editors. Just now I was struggling with a merge using the VS Code editor and found myself wishing I had a base view. So then I searched to see whether it was possible, and was very surprised (and happy) to learn there's an option for it! I definitely would have turned it on if I'd known about it.

**Comment by yairchu** [comment]
> To them, the base view is not as useful.

@hediet , you hold a misconception the the base view isn't helpful.

Let me try to illustrate why it's absolutely critical to have.

In the following conflict resolution quiz, most people choose option B as the correct resolution:

<img width="716" height="417" alt="Image" src="https://github.com/user-attachments/assets/4102a552-311c-43ec-89b8-f8e2f7f787e9" />

But when presented with the base people understand that option C was the right answer:

<img width="215" height="446" alt="Image" src="https://github.com/user-attachments/assets/97b2d5e2-7afa-455e-a6ac-a1c9e1a017ac" />

They only went for option B because people naturally assume that code is always added and never deleted, but when they see the base they understand that isn't the case.

For a more elaborate explanation see https://www.youtube.com/watch?v=c3eV5HVdUuk
